kubelet/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG - kubelet
 
+## 2.3.0 / 2018-11-30
+
+* [Added] Add restart and container state metrics to kubelet. See [#2605](https://github.com/DataDog/integrations-core/pull/2605). Thanks [schleyfox](https://github.com/schleyfox).
+* [Added] [kubelet] add more cpu metrics. See [#2595](https://github.com/DataDog/integrations-core/pull/2595).
```

Comment:
Wondering if we can cleanup these lines to remove `kubelet` since its the changelog specifically for this check.